### ISSUE TITLE
Alert pages very slow to process when first opened.

### ISSUE THREAD

**Issue by seanflynncontact** [Sitata/issue_tracking#23]
Hospitals etc are very slow to finish syncing/processing when first opened.

**Comment by johnmeade** [comment]
The sync process between the Sitata SDK and Sitata servers takes a couple seconds on first launch. It also occurs with a rough ordering, ie trips and alerts are downloaded first, which triggers live updates in the GUI when they are saved, and lower priority items are downloaded afterwards. Hospitals / medications / vaccinations / diseases are downloaded just after the trips and alerts. Is the delay you are seeing more than, say, 5-10 seconds?

**Comment by seanflynncontact** [comment]
Yes it often is more than 5 to 10 seconds. In fact I made a new profile user and synced. There destination is France. I just clicked Hospitals and its been spinning with no map for several minutes, I noticed this earlier also, Also I upped the timeout you suggested when clicking trip advice to 30 seconds, I still often times out on 1st click but then works the second time. This has been the consistent issue going back to the empty destination problem. Its better because now it waits but the underlying failing issue is still there,

**Comment by johnmeade** [comment]
When using our (slow) staging server, all the relevant data should be loaded in about 5 seconds if there are no errors. Could you post the android logs for this issue as well?

**Comment by seanflynncontact** [comment]
[appOutputLogs.txt](https://github.com/Sitata/issue_tracking/files/1493049/appOutputLogs.txt)

**Comment by seanflynncontact** [comment]
John. I added the output log. Hospitals don't load for the following user in staging. The following are the credentials.

one1@test.com
Travelinsured01

**Comment by johnmeade** [comment]
I don't see anything in logs that would cause this behaviour... This is a shot in the dark, but could you try changing the way you are starting jobs to the following:
```csharp
// old:
BaseApp.GetSitataGui().Ctrlr.Jobs.X.Y().ReplaceAndStart();
BaseApp.GetSitataGui().Ctrlr.Jobs.X.Y().FindOrStart(); // <-- you probably aren't using this one anyway

// new:
BaseApp.GetSitataGui().Ctrlr.Jobs.X.Y().Start();
```
for all `X` and `Y` everywhere in your app?
